### DISCUSSION TITLE
How to download  Azure Universal Packages Artifact.

### DISCUSSION THREAD

**Discussion by santhoshmohanr** [cake-build/cake#3657] [Q&A]
I need to download Azure Universal Packages Artifact and need to use it in builds. I can able to download Azure Pipeline associated artifact using this code as shows in this example (https://cakebuild.net/api/Cake.AzureDevOps/AzureDevOpsAliases/238B1087). 
But I need to download Azure Universal Packages Artifact which is not associated with Azure Pipeline. 

Please help here to clarify this doubts.

**Comment by JuergenRB** [comment]
It is possible to download the Universal Artifacts with the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/artifacts/universal?view=azure-cli-latest#az_artifacts_universal_download). So I think you should use [Cake.AzureCli](https://www.nuget.org/packages/Cake.AzureCli/).

However, I can't find Az().Artifacts 
![image](https://user-images.githubusercontent.com/22403655/144027921-3599bd01-4b4c-4f20-aedf-f2664ecbef6f.png)

maybe @lprichar  can help.
